modules/ocf/manifests/systemd/service.pp

```diff
@@ -3,22 +3,27 @@
   $content = undef,
   $ensure = 'running',
   $enable = true,
+  $suffix = 'service',
 ) {
-  file { "/etc/systemd/system/${title}.service":
+  if $content and $source {
+    fail('You may not supply both content and source parameters to ocf::systemd::service')
+  }
+  elsif $content == undef and $source == undef {
+    fail('You must supply either the content or source parameter to ocf::systemd::service')
+  }
+
+  file { "/etc/systemd/system/${title}.${suffix}":
     source   => $source,
     content  => $content,
     notify   => Exec['systemd-reload'],
-    require  => Package['systemd-sysv'],
-  }
-
-  service { $title:
+  }~>
+  service { "${title}.${suffix}":
     ensure    => $ensure,
     enable    => $enable,
     provider  => systemd,
-    subscribe => File["/etc/systemd/system/${title}.service"],
```

Comment:
When the service's unit file is changed, doesn't removing this mean that it won't restart the service at all any more?

Reply:
I changed that to a notify arrow, ```~>```, because it defines that relationship (I think) in a much cleaner manner.

Reply:
Ah yeah, so you did. I'm not very used to using that format, this looks good then.